src/commcare_cloud/fab/utils.py

```diff
@@ -153,15 +153,15 @@ def diff(self):
         return DeployDiff(self.repo, self.last_commit_sha, self.deploy_ref)
 
 
-GITHUB_CREDENTIALS = None
+GITHUB_TOKEN = None
 
 
-def _get_github_credentials(message, force=False):
-    global GITHUB_CREDENTIALS
+def _get_github_token(message, force=False):
+    global GITHUB_TOKEN
 
-    if GITHUB_CREDENTIALS is not None:
-        if not force or GITHUB_CREDENTIALS[0]:
-            return GITHUB_CREDENTIALS
+    # return existing token if it exists and caller has not requested a fresh token fetch
```

Comment:
see main comment for details but a 'fresh token fetch' isn't required here - only a non-empty token.